atomic

```diff
@@ -389,13 +392,13 @@ if __name__ == '__main__':
     scanp = subparser.add_parser(
         "scan", help=_("scan an image or container for CVEs"),
         epilog="atomic scan <input> scans a container or image for CVEs")
-    scanp.set_defaults(func='scan')
+    scanp.set_defaults(_class=Scan, func='scan')
     scan_group = scanp.add_mutually_exclusive_group()
     scan_out = scanp.add_mutually_exclusive_group()
-    scan_out.add_argument("--json", default=False, action='store_true', help=_("output json"))
-    scan_out.add_argument("--detail", default=False, action='store_true', help=_("output more detail"))
     scanp.add_argument("scan_targets", nargs='*', help=_("container image"))
     scanp.add_argument("--fetch_cves", type=bool_detect, default=None,  help=_("override the behavior defined in /etc/oscapd/config.ini as to whether to download the most recent CVE data. Values can be True of False"))
+    scanp.add_argument("--scanner", choices=[x for x in atomic_config['scanners']], default='openscap', help=_("define the intended scanner"))
```

Comment:
Not sure we should have --fetch_cves any longer, this is specific to one container type.  Should be inside the container.